tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/InitConfigureProjectPage.java

```diff
@@ -308,32 +303,32 @@ public void addPortalRuntimeAndServer( String serverRuntimeName, String location
 
     }
 
-    private void backupSDK( IProgressMonitor monitor )
+    private void backup( IProgressMonitor monitor )
     {
-        Boolean backupSdk = dataModel.getBackupSdk().content();
+        Boolean backup = dataModel.getBackupSdk().content();
 
-        if ( backupSdk == false )
+        if ( backup == false )
         {
             return;
         }
 
         SubMonitor progress = SubMonitor.convert( monitor, 100 );
         try
         {
-            progress.setTaskName( "Backup sdk folder into Eclipse workspace...");
-            org.eclipse.sapphire.modeling.Path originalSDKPath = dataModel.getSdkLocation().content();
+            progress.setTaskName( "Backup origial project folder into Eclipse workspace...");
```

Comment:
We should let them modify the location of where the backup zip is going to be stored.  But the default would be the Eclipse workspace root location.